index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header class="toolbar_container">
+			<a href="#" class="Title_logo">My Logo</a>
+			<nav>
+				<img src="./images/i.png"  class="logo" alt="Icon menu">
+			</nav>
+		</header>
+
+		<main class="main_container">
+			<section class="text_container">
+				<h1 class="paragraphe">Hey There. <br/>I’m Leader.</h1>
+				
+				<h4>I'm a Software Developer</h4>
+				<p class="para">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+					don’t hesitate to contact me.</p>
+			</section>
+			
+			<section class="icons-container">
+				<ul class="icons-containers">
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon -GitHub.png"/></li>
+					<li><img src="./images/Icon - Linkedin.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Enabled.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Twitter.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Medium.png"/></li>	
+				</ul>				
+			</section>
+		</main>
+	</section>
+	
+	<section class="second_container">
+		<div class="header_title">
+		<h2>My Recent Works</h2>
+		<p class="line"></p>
+		</div>
+		<div class="recent_work_container">
+			<img src="./images/imgs.png" alt="works_section">
+			<div class="recent_work_container_text">
+				<h3>Multi-Post Stories</h3>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+			<div class="butons_container">
+				<button type="button">CSS</button>
+				<button type="button">HTML</button>
+				<button type="button">Boostrap</button>
+				<button type="button">Ruby</button>
+			</div>
+			<button  type="button" class="project_btn">See project</button>
+			</div>
+		</div>
+	</section>
```

Comment:
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/253671263-1c2ec42b-efe1-4749-8890-b0938bf052ac.png)|![image](https://user-images.githubusercontent.com/99715640/253671327-e9b1ae7e-3e62-4c2f-9485-67a7404e03d6.png)|

- [ ] Kindly observe and fix the disparities that we have above in terms of:
  - [ ] Straying social icons
  - [ ] color of your `project title`, `paragraph`, and `list of technologies`
  - [ ] font-weight of the list of technologies
  - [ ] the thickness of the divider (horizontal line)
  - [ ] your button is not well aligned to the center